package.json

```diff
@@ -117,7 +117,6 @@
       "remove-flow-types-loader>loader-utils": "*"
     },
     "patchedDependencies": {
-      "react-native-fast-crypto@2.2.0": "patches/react-native-fast-crypto@2.2.0.patch",
       "react-native-video@5.2.0": "patches/react-native-video@5.2.0.patch",
```

Comment:
is there any reason we remove this ? or is it not just an up to the latest develop bug ? 🤔

Reply:
I think you are right; I will add it back!